Gemfile

```diff
@@ -85,6 +85,7 @@ gem 'directo', github: 'internetee/directo', branch: 'master'
 
 group :development, :test do
   gem 'pry', '0.14.1'
+  gem 'rubocop'
```

Comment:
It is better to add the rubocop gem in a separate branch, add general settings for the rubocop, and also with the consent of all other developers.

Reply:
I added rubocop only for development (and test) for manually checking files before commit.